sound/soc/sof/topology.c

```diff
@@ -2310,16 +2310,6 @@ static int sof_complete(struct snd_soc_component *scomp)
 	struct snd_sof_route *sroute;
 	int ret;
 
-	list_for_each_entry(sroute, &sdev->route_list, list) {
-		ret = sof_widget_update_ipc_route(scomp, sroute);
-		if (ret < 0) {
-			dev_err(sdev->dev, "failed updating IPC struct for route %s -> %s\n",
-				sroute->src_widget->widget->name,
-				sroute->sink_widget->widget->name);
-			return ret;
-		}
-	}
```

Comment:
The reason that this can be removed is that we have dynamic pipelines and the route is set up when the stream is starting? No need (and point) to set up routes at the end of topology loading?

Reply:
yes exactly